setup.cfg

```diff
@@ -14,6 +14,8 @@ url = https://github.com/pykeen/pykeen
 download_url = https://github.com/pykeen/pykeen/releases
 project_urls =
     Bug Tracker = https://github.com/pykeen/pykeen/issues
+    Documentation = https://pykeen.readthedocs.io
+    Homepage = https://pykeen.github.io
```

Comment:
Did this part end up in the wrong branch? :sweat_smile:

Reply:
I didn’t know where to start but I wanted to make a branch and have CI run